package.json

```diff
@@ -221,6 +222,7 @@
     "eslint-plugin-react": "^7.20.6",
     "eslint-plugin-react-hooks": "^4.0.8",
     "file-loader": "^6.0.0",
+    "husky": "^4.2.5",
```

Comment:
isnt the latest version like 7? I remember there was some chat about husky, but cant recall details.

Reply:
yup, but our setup is for 4.2.5. I think we should create a separate ticket to upgrade husky + general pre-hook commits